core/meson.build

```diff
@@ -40,46 +40,33 @@ install_subdir('vibe/', install_dir: 'include/d/vibe/')
 # Build Targets
 #
 
-core_link_with = [vibe_utils_lib,
-                  vibe_data_lib]
-
 # Basic I/O and concurrency primitives, as well as low level utility functions
 vibe_core_lib = library('vibe-core',
         [vibe_core_src],
-        include_directories: [vibe_utils_src_dir,
-                              vibe_data_src_dir,
-                              openssl_inc,
-                              libevent_inc],
         install: true,
-        dependencies: [crypto_dep,
-                       ssl_dep,
-                       libevent_dep,
-                       zlib_dep],
-        link_with: [core_link_with],
+        dependencies: [vibe_utils_dep, vibe_data_dep],
         version: project_version,
         soversion: project_soversion
 )
 pkgc.generate(name: 'vibe-core',
-              libraries: [vibe_core_lib] + core_link_with,
```

Comment:
This should instead gain a `requires: ['vibe-utils', 'vibe-data']` line to ensure that if someone links a project with vibe-core, those two get linked to as well (since pretty much all of vibe-core depends on utils and data in its public routines).
This applies to all of these instances.

Reply:
Ok. Done for all meson files.

Reply:
Thanks a lot! I haven't tested this yet, but the Meson files themselves look fine to me, even much better than before :-)